GW2EIEvtcParser/EIData/ProfHelpers/Mesmer/MesmerHelper.cs

```diff
@@ -323,5 +323,39 @@ internal static bool IsKnownMinionID(long id)
         {
             return NonCloneMinions.Contains(id) || IsClone(id);
         }
+        
+        internal static void ComputeProfessionCombatReplayActors(AbstractPlayer player, ParsedEvtcLog log, CombatReplay replay)
+        {
+            // TODO: tracking ids seem broken in arc? using buff remove with hardcoded fallback for now
+            foreach (EffectEvent effect in ProfHelper.GetEffectsForPlayer(log.CombatData, player, EffectGUIDs.MesmerPortalInactive))
+            {
+                int start = (int)effect.Time;
+                var remove = log.CombatData.GetBuffData(PortalWeaving).OfType<AbstractBuffRemoveEvent>().FirstOrDefault(x => x.Time >= start);
+                int end = (int?)remove?.Time ?? start + 60000;
+                replay.Decorations.Add(new IconDecoration("https://wiki.guildwars2.com/images/8/81/Portal_Entre.png", 128, 0.5f, effect.Src.ID, (start, end), new PositionConnector(effect.Position)));
```

Comment:
Src.UniqueID, ID is the SpeciesID

Reply:
You should simply give an AgentItem to the Constructor and have the constructor put the proper value on the attribute.